contrib/eigen/template.py

```diff
@@ -0,0 +1,17 @@
+pkgname = "eigen"
+pkgver = "3.4.0"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = ["cmake", "ninja", "pkgconf"]
+pkgdesc = "C++ template library for linear algebra"
+maintainer = "shtayerc <david.murko@mailbox.org>"
+license = "BSD-3-Clause"
+url = "https://gitlab.com/libeigen/eigen"
+source = f"{url}/-/archive/{pkgver}/eigen-{pkgver}.tar.bz2"
+sha256 = "b4c198460eba6f28d34894e3a5710998818515104d6e74e5cc331ce31e46e626"
+# no check target
```

Comment:
technically there is if you enable them, though i'm not sure how much point there is in testing a header-only library. maybe just update the comment?